### PR TITLE
MirageSDK: Use newer dhcp client

### PR DESCRIPTION
**Pull request by yomimono**
**- What I did**

Changed the MirageSDK dhcp client example to use a branch of `charrua-client` with a better API for MirageSDK.

**- A picture of a cute animal (not mandatory but encouraged)**
whale_cat

### CODE REVIEW

File: projects/miragesdk/examples/mirage-dhcp.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ services:
   - name: rngd
     image: mobylinux/rngd:3dad6dd43270fa632ac031e99d1947f20b22eec9
   - name: dhcp-client
-    image: mobylinux/dhcp-client:a7a6b49b0ff51ffa2f44ac848cd649e29f946e0c
+    image: mobylinux/dhcp-client:24e63a65d9a7a20f0f9b3b10e86eeb07b3e26427
```
Comment (justincormack) [inline projects/miragesdk/examples/mirage-dhcp.yml]: we shouldnt be using `mobylinux` any more. We have a `linuxkitprojects` now I can give you access

Comment (rn) [inline projects/miragesdk/examples/mirage-dhcp.yml]: can we change the `mobylinux` bits. For standard packages we should use the `linuxkit` org and for projects stuff we have set up the `linuxkitprojects` org.

Comment (yomimono) [inline projects/miragesdk/examples/mirage-dhcp.yml]: cool, please give it to docker hub username `yomimono` rather than `mindypreston`

Comment (justincormack) [inline projects/miragesdk/examples/mirage-dhcp.yml]: ok you should ahve push access to `linuxkitprojects/dhcp-client` now

**Pull request merged**